sklearn/multiclass.py

```diff
@@ -203,6 +215,53 @@ def fit(self, X, y):
 
         return self
 
+    def partial_fit(self, X, y, classes=None):
+        """Partially fit underlying estimators
+        
+        Should be used when memory is inefficient to train all data.
+        Chunks of data can be passed in several iteration.
+        
+        Parameters
+        ----------
+        X : (sparse) array-like, shape = [n_samples, n_features]
+            Data.
+        y : (sparse) array-like, shape = [n_samples] or [n_samples, n_classes]
+            Multi-class targets. An indicator matrix turns on multilabel
+            classification.
+        classes : array, shape (n_classes,)
+            Classes across all calls to partial_fit.
+            Can be obtained via `np.unique(y_all)`, where y_all is the
+            target vector of the entire dataset.
+            This argument is only required in the first call of partial_fit
+            and can be omitted in the subsequent calls.
+        Returns
+        -------
+        self
+        """
+        if _check_partial_fit_first_call(self, classes):
+            self.estimators_ = [clone(self.estimator) for _ in range
+                                (self.n_classes_)]
+        
+        # A sparse LabelBinarizer, with sparse_output=True, has been shown to
+        # outpreform or match a dense label binarizer in all cases and has also
+        # resulted in less or equal memory consumption in the fit_ovr function
+        # overall.
+        self.label_binarizer_ = LabelBinarizer(sparse_output=True)
+        Y = self.label_binarizer_.fit_transform(y)
+        Y = Y.tocsc()
+        columns = (col.toarray().ravel() for col in Y.T)
+
+        self.estimators_ = Parallel(n_jobs=self.n_jobs)(delayed(
+            _partial_fit_binary)(self.estimators_[i],
+            X, next(columns) if self.classes_[i] in
+            self.label_binarizer_.classes_ else
+            np.zeros((1, len(y))), classes =
+                     ["not %s" % self.classes_[i],
```

Comment:
I think you can just do away with the classes argument, unless you want to warn the user that a certain class is not present during a certain batch_fit.